webapp/client/src/components/ButtonLink.js

```diff
@@ -0,0 +1,53 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+const Anchor = styled.a`
+  padding: 1rem 1.25rem calc(1rem - 4px) 1.25rem;
+  margin-right: var(--spacing-05);
+  font-size: var(--text-base);
+  letter-spacing: var(--tracking-wide);
+  text-decoration: none;
+  color: var(--inverse-text-color);
+  background: var(--link-color);
+  display: inline-block;
+  position: relative;
+  border: none;
+  border-radius: 0;
+  border-bottom: 4px solid var(--link-color);
+
+  &:not(:disabled):not(.disabled):active {
+    background: var(--link-color) !important;
+    border: none !important;
+    border-bottom: 4px solid var(--link-color) !important;
+  }
+
+  &:hover {
+    background: var(--link-hover-color);
+    border: none;
+    border-bottom: 4px solid var(--link-hover-color);
+    color: var(--inverse-text-color);
+    text-decoration: none;
+  }
+
+  &:focus {
+    color: var(--focus-text-color);
+    background: var(--focus-color);
+    outline: none;
+    box-shadow: none;
+    border: 0;
+    border-bottom: 4px solid var(--focus-border-color);
+  }
+
+  &:visited {
+    color: var(--inverse-text-color);
+  }
+`;
+
+export default function ButtonLink({ text, url }) {
+  return <Anchor href={url}>{text}</Anchor>;
+}
+
+ButtonLink.propTypes = {
+  text: PropTypes.string.isRequired,
+  url: PropTypes.string.isRequired,
+};
```

Comment:
There is already a component Download link, which is the same like this one. Maybe it would be a good idea to expand the existing one with a prop to render the download attribute or not and rename it, e.g. anchor component or so.

Reply:
Done :) we now have a AnchorButton with a Download Option